src/components/common/RaydiantIcon/index.tsx

```diff
@@ -0,0 +1,25 @@
+import { FC } from 'react';
+
+import { AddIcon } from './AddIcon';
+import { CloseIcon } from './CloseIcon';
+import { PlaylistsIcon } from './PlaylistsIcon';
+import { PlaylistAddIcon } from './PlaylistAddIcon';
+
+export type IconName = 'close' | 'add' | 'playlists' | 'playlist-add';
+
+const iconMap = {
+  close: CloseIcon,
+  add: AddIcon,
+  playlists: PlaylistsIcon,
+  'playlist-add': PlaylistAddIcon,
+};
+
+export const RaydiantIcon: FC<{ name: IconName }> = ({ name }) => {
+  const IconComponent = iconMap[name];
+  return (
+    // Negative margin ensures icon is vertically centered with text
+    <span className="relative top-[6px] inline-flex">
```

Comment:
Currently, all Raydiant icons are used inline with text. I decided to isolate the Raydiant icons in a new file `RaydiantIcons` instead of extending our existing `Icons` component. That allowed me to apply this negative margin to all our Raydiant icons.